docs/docs/blocks/block-factories.mdx

```diff
@@ -547,7 +547,7 @@ export function OptionalImageBlock({ data }: PropsWithData<OptionalImageBlockDat
 The RichTextBlock factory can be used to create a block with a rich text.
 It requires a link block to be set.
 
-import RichTextBlock from "./images/rich-text-block.png";
+import { RichTextBlock } from "./images/rich-text-block.png";
```

Comment:
I doubt that the image will work when imported like this.

Reply:
Renamed too much.